src/controller/java/src/chip/onboardingpayload/OnboardingPayload.kt

```diff
@@ -55,6 +102,18 @@ class OnboardingPayload(
 ) {
   var optionalQRCodeInfo: HashMap<Int, OptionalQRCodeInfo>
 
+  enum class VendorId(val value: Int) {
+    COMMON(0x0000),
+    UNSPECIFIED(0x0000),
+    APPLE(0x1349),
+    GOOGLE(0x6006),
```

Comment:
should we try to be more comprehensive here based on https://github.com/project-chip/connectedhomeip/blob/master/src/app/zap-templates/zcl/data-model/manufacturers.xml ?